src/partials/messenger.ts

```diff
@@ -94,22 +98,26 @@ class SendFileController extends DialogController {
  * Handle device unreachable
  */
 export class DeviceUnreachableController extends DialogController {
-    public static readonly $inject = [
-        '$rootScope', '$window', '$mdDialog',
-        'StateService', 'WebClientService',
-    ];
     private readonly $rootScope: any;
     private readonly $window: ng.IWindowService;
     private readonly stateService: StateService;
     private readonly webClientService: WebClientService;
     public retrying: boolean = false;
     public progress: number = 0;
 
+    public static readonly $inject = [
```

Comment:
Nit-Question: Shouldn't we apply the same logic here regarding "if it doesn't fit a line, use one line per parameter"?